public/editor-client/src/types/AiText.ts

```diff
@@ -9,25 +8,61 @@ export enum Role {
   Assistant = "assistant",
 }
 
-export interface Conversation {
-  role: Role;
-  content: string;
-}
-
 export enum Action {
   Extend = "extend",
   Shorten = "shorten",
   Simplify = "simplify",
-  English = "English",
-  German = "German",
-  French = "French",
-  Spanish = "Spanish",
-  Italian = "Italian",
-  Dutch = "Dutch",
-  Portuguese = "Portuguese",
-  Polish = "Polish",
   Professional = "Professional",
   Casual = "Casual",
   Confident = "Confident",
   Friendly = "Friendly",
 }
+
+interface ConversationMessage {
+  role: Role;
+  content: string;
+}
+
+export interface Conversation {
+  temperature: number;
+  messages: ConversationMessage[];
+}
+
+export const LanguageCodes: Record<string, string> = {
```

Comment:
this is not full list of languages codes conform https://www.npmjs.com/package/franc#data why is only this ?